index.js

```diff
@@ -37,13 +40,16 @@ module.exports = {
 				ignore: ['after-comment', 'inside-single-line-block'],
 			},
 		],
+		'custom-media-pattern': /^([a-z][a-z0-9]*)(-[a-z0-9]+)*$/,
```

Comment:
I think all rules with RegExps would be confusing for non-power user. Because if there is a violation we will show this RegExp and use might not be familiar with it at all.

Reply:
Good point.

We can use the [`message` secondary option](https://stylelint.io/user-guide/configure/#message) to make them simpler, e.g.:

> Expected custom media name to be in kebab case, e.g. "custom-media-name"

Reply:
We could try that.

Reply:
Done. Each is slightly adapted from the original message in the rule.